docs/topics/con_how-caching-works-and-when-you-need-it.adoc

```diff
@@ -0,0 +1,33 @@
+[id='how-caching-works-and-when-you-need-it_{context}']
+= How caching works and when you need it
+
+Caching allows you to store information and access it for a given period of time.
```

Comment:
That sounds weird. _Caches_ allow you to store information etc. _Caching_ is a process of storing information etc.

Reply:
OK, I'll change that.